testsuite/tests/custom_tenant/test_custom_tenant.py

```diff
@@ -37,6 +37,9 @@ def get_custom_client(url, access_token, ssl_verify):
 
 # FIXME: threescale_api.errors.ApiClientError: Response(422): b'{"errors":{"system_name":["must be shorter."]}}
 # Name of the tenant is too long
+# ^ no longer occuring
+# FIXME: threescale_api.errors.ApiClientError: Response(404 Not Found): b'{"status":"Not found"}'
```

Comment:
Would it be possible to override `custom_application` just for this test to add backoff there too?

Reply:
I would discuss it in https://github.com/3scale-qe/3scale-tests/pull/57, what do you think?

Reply:
Well, not sure if this should be testsuite wise, I would say this is only for this test